### DISCUSSION TITLE
Converting gromacs files with multiple moleculetypes

### DISCUSSION THREAD

**Discussion by lukekruse** [hernanchavezthielemann/GRO2LAM#42] [Q&A]
Hello!
I am trying to convert my gromacs simulation files to lammps with your script but I have multiple unique proteins in my simulation. My topology file contains a line of '#include "Protein_1.itp"' for each protein in the simulation. How can I modify my topology file so that I can use your tool?
Thanks!
Luke

**Comment by hernanchavezthielemann** [comment]
Hi @Entropy134,

for some reason I did not receive any warning about this message.
Gro2lam should work fine, it accepts #includes, so:
load the gro and top files, and then press autoload, finally try to convert your system.

best,
Hernan
